magento/ece-tools
# MAGECLOUD-2886: [Cloud Docker] Add Redis 3.2 and 4.0

### Fixed Issues (if relevant)
https://magento2.atlassian.net/browse/MAGECLOUD-2886

### Manual testing scenarios

1. Generate config with option `--redis {3.2, 4.0}`
2. Run Docker env
3. Verify Magento works with Redis cache

### Contribution checklist
